etc/fetchers/fetch-faq-sheets.js

```diff
@@ -7,14 +7,21 @@ const FAQ_LINK = "https://kcov.id/wbw-faq";
 module.exports.fetchFaqSheets = async function fetchFaqSheets() {
   const source = await fetch(FAQ_LINK);
   const $ = cheerio.load(await source.text());
+  $.prototype.textln = function () {
+    this.find("br").replaceWith("\n");
+    this.find("*").each(function () {
+      $(this).replaceWith($(this).html());
+    });
+    return this.html();
+  };
```

Comment:
If we decided to go with `htmr` (as mentioned above), these lines are no longer necessary.